src/pages/home/report/ReportActionsView.js

```diff
@@ -51,6 +52,7 @@ class ReportActionsView extends React.Component {
 
         this.state = {
             refetchNeeded: true,
+            activeReportActionID: null,
```

Comment:
Wondering if making this a state variable is the best idea. If we update this then it will re-render the `FlatList` which we should try to do as infrequently as possible. Have you tested this against a production report with a lot of chat history to see what happens?